server/controllers/user.js

```diff
@@ -63,8 +63,8 @@ function getGroupsForUser(req, res, next) {
     .catch(err => next(err));
 }
 
-function loadUser(req, res, next, userId) {
-  userService.getUser(userId)
+function loadUser(req, res, next) {
```

Comment:
I added this to a separate card, because it has more to do with Authentication than a user leaving a group